### ISSUE TITLE
Welcome cover needs click/tap to slide up

### ISSUE THREAD

**Issue by SiravitPhokeed** [Emocial-NLP-Depression-Detection/Emocial-Frontend#5]
**Seen in**: /
**Affects**: All devices
**Intended behaviour**: Welcome cover should be selectable by using the tab key, or should slide up on any key press
